Sources/SwifterSwift/UIKit/UINavigationControllerExtensions.swift

```diff
@@ -31,6 +31,18 @@ public extension UINavigationController {
         pushViewController(viewController, animated: true)
         CATransaction.commit()
     }
+    
+    /// SwifterSwift: Pushes a view controller while hiding or showing the bottom bar.
+    ///
+    /// - Parameters:
+    ///   - viewController: The view controller to push.
+    ///   - hidesBottomBar: If `true`, hides the bottom bar (e.g. tab bar).
+    ///   - animated: Specify `true` to animate the transition.
+    
+    func pushViewController(_ viewController: UIViewController, hidesBottomBar: Bool, animated: Bool) {
```

Comment:
Remove extra empty line
```suggestion
    ///   - animated: Specify `true` to animate the transition.
    func pushViewController(_ viewController: UIViewController, hidesBottomBar: Bool, animated: Bool) {
```